lib/errors.js

```diff
@@ -516,6 +523,25 @@ function createUnparsableFileError(message, filename) {
   return err;
 }
 
+/**
+ * Creates an error object to be thrown when file attempts to import module that does not exist.
+ * @public
+ * @static
+ * @param {string} message - Error message to be displayed.
+ * @param {string} importerFilename - File name of importing module
+ * @param {string} importedFilename - File name of imported module
+ * @returns {Error} Error with code {@link constants.MODULE_NOT_FOUND}
+ */
+function createModuleNotFoundError(
```

Comment:
We don't need a new Mocha error code, we can't wrap each and every Node error in a new Mocha error object. We just have to rethrow Node's error somehow or include Node's error message in Mocha's `UnparsableFileError` object.